cmd/infra/azure/create.go

```diff
@@ -558,38 +555,46 @@ func createRhcosImages(ctx context.Context, l logr.Logger, o *CreateInfraOptions
 	if storageAccountKeyResult.Keys == nil || len(storageAccountKeyResult.Keys) == 0 || storageAccountKeyResult.Keys[0].Value == nil {
 		return "", errors.New("no storage account keys exist")
 	}
-	blobAuth, err := autorest.NewSharedKeyAuthorizer(storageAccountName, *storageAccountKeyResult.Keys[0].Value, autorest.SharedKey)
+
+	credential, err := container.NewSharedKeyCredential(storageAccountName, *storageAccountKeyResult.Keys[0].Value)
 	if err != nil {
-		return "", fmt.Errorf("failed to construct storage object authorizer: %w", err)
+		return "", err
 	}
 
-	blobClient := blobs.New()
-	blobClient.Authorizer = blobAuth
-	l.Info("Uploading rhcos image", "source", sourceURL)
-	input := blobs.CopyInput{
-		CopySource: sourceURL,
-		MetaData: map[string]string{
-			"source_uri": sourceURL,
-		},
+	imageBlobURL := "https://" + storageAccountName + ".blob.core.windows.net/" + "vhd" + "/"
```

Comment:
nit: you don't need to concat literals. It would also be cleaner with fmt.Sprintf

Reply:
also, should this variable be named `imageBlobURLPrefix` ?